### PR TITLE
fix: local state storage content simplified

### PR DESCRIPTION
**Pull request by raducrisan1**

### CODE REVIEW

File: ImmuDB4Net/ImmuStateSerde.cs [hunk 1]
```diff
@@ -35,26 +34,13 @@ public class ImmuStateSerde
         if (session == null)
             return null;
         string contents = File.ReadAllText(fileName);
-        var deserialized = JsonSerializer.Deserialize<Dictionary<string, ImmuState>>(contents);
```
Comment (byo) [inline ImmuDB4Net/ImmuStateSerde.cs:38]: Wouldn't it be easier to move the json serialization / deserialization code directly to the fille state handler?

If you think serde is still a valid addition, then the `Read` method should not get the `database` nor `session` parameters.

File: ImmuDB4Net/ImmuStateSerde.cs [hunk 2]
```diff
@@ -35,26 +34,13 @@ public class ImmuStateSerde
...
-        string key = session.ServerUUID + "_" + database;
-        if (statesMap.TryGetValue(key, out var state))
-        {
-            return state;
-        }
-        return null;
+        return JsonSerializer.Deserialize<ImmuState>(contents);
     }
 
     public void Write(string fileName, Session session, ImmuState state)
```
Comment (byo) [inline ImmuDB4Net/ImmuStateSerde.cs:40]: `session` parameter should be removed

Comment (raducrisan1) [reply ImmuDB4Net/ImmuStateSerde.cs:40]: will remove the class entirely, as you suggested. I also think it's better, in fact have already discussed about it.

File: ImmuDB4Net/ImmuStateSerde.cs (hunk 1 above)
Comment (raducrisan1) [reply ImmuDB4Net/ImmuStateSerde.cs:38]: it's easier indeed just to get rid of immustateserde. I have just removed it.

**Review by byo — APPROVED** [review]

**Pull request merged**